__sandbox__/Sandbox.tsx

```diff
@@ -90,11 +90,26 @@ export function Sandbox(): any {
           HOOK_HELPER,
         })
         if (useWindow) {
-          session.openInNewWindow()
+          session.openInNewWindow({
+            customFields: [
```

Comment:
I think we are not passing this new field when we launch an import session using Flatfile.startOrResumeImportSession.